jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcExecutionRepository.java

```diff
@@ -295,9 +295,13 @@ private SelectConditionStep<Record1<Object>> findSelect(
                 field("value")
             )
             .from(this.jdbcRepository.getTable())
-            .where(this.defaultFilter(tenantId, false))
-            .and(NORMAL_KIND_CONDITION);
+            .where(this.defaultFilter(tenantId, false));
 
+        boolean hasKindFilter = filters != null && filters.stream()
+            .anyMatch(f -> "kind".equalsIgnoreCase(f.field().name()) );
```

Comment:
```suggestion
            .anyMatch(f -> KIND.value()..equalsIgnoreCase(f.field().name()) );
```

This to avoid hardcoding the value of the KIND enum item

Reply:
done